src/datatypes/point.c

```diff
@@ -19,3 +19,17 @@ float Point_lon(SIValue point) {
 	return point.point.longitude;
 }
 
+SIValue Point_GetCoordinate(SIValue point, SIValue key) {
+	ASSERT(SI_TYPE(point) == T_POINT);
+	ASSERT(SI_TYPE(key) == T_STRING);
+
+	if(strcmp(key.stringval, "latitude")==0) {
```

Comment:
should we use case agnostic compare ?
is `LatiTude` a valid string in this case ?

Reply:
I think that `LatiTude` should be invalid.
The comparison is case sensitive because it was done based on [Map_KeyIdx](https://github.com/RedisGraph/RedisGraph/blob/7765354206f2a35c8b325f1f7c90c588fae2c0bc/src/datatypes/map.c#L56), the keys in the maps are case sensitive.